_includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md

```diff
@@ -114,7 +116,7 @@ Make sure that the following changes have been applied in the database:
 | spy_region.uuid                     | column | created |
 | spy_sales_order_item_service_point  | table  | created |
 
-Make sure that propel entities have been generated successfully by checking their existence. Also, make generated entity classes extending respective Spryker core classes.
+Make sure propel entities have been generated successfully by checking their existence. Also, make generated entity classes extending  respective Spryker core classes.
```

Comment:
Could you elaborate on the second sentence?

Reply:
I guess it should be general verification for any Integration Guide that contains new tables. So it's better to keep consistency with other IGs.

Reply:
Do you mean I should remove the second sentence?

Reply:
nope please keep it, there are 2 possible verifications:
- We added new propel tables, columns: 
 ```
 Make sure propel entities have been generated successfully by checking their existence. Also, make generated entity classes extending respective Spryker core classes.
 ```
- We added ONLY tables:
```
Make sure propel entities have been generated successfully by checking their existence.
```